demisto_sdk/commands/download/downloader.py

```diff
@@ -236,6 +238,62 @@ def download_playbook_yaml(self, playbook_string) -> str:
 
         return playbook_string
 
+    def handle_incidentfield(self, incidentfield_string: str, scripts_mapper: dict) -> str:
+        # In case the incident field uses a custom script, replace the id value of the script with its name
+        file_json_object = json.loads(incidentfield_string)
+        incidentfield_name = file_json_object.get('name')
+        if incidentfield_name and (incidentfield_name in self.input_files or self.all_custom_content):
+            if file_json_object.get('script') and file_json_object.get('script') in scripts_mapper:
+                incidentfield_string = incidentfield_string.replace(file_json_object.get('script'),
+                                                                    scripts_mapper[file_json_object.get('script')])
```

Comment:
please replace the multiple `.get('script')`s with a single call saving its value as a variable.